### DISCUSSION TITLE
Error:git submodule exited with code 128

### DISCUSSION THREAD

**Discussion by highHumann** [Cyfrin/foundry-full-course-cu#264] [General]
I am getting this error while running forge init 

![Screenshot (47)](https://github.com/Cyfrin/foundry-full-course-f23/assets/122611774/63b94311-5e47-47b2-bb0a-56b60eea18c6)

**Comment by highHumann** [comment]
it can be resolve by running
```
git submodule sync
git submodule update --init --recursive
```
in the terminal

**Comment by drained69** [comment]
good day, please how did you fix this?
